FLEDGE.md

```diff
@@ -297,6 +316,8 @@ else
 
 This will cause the browser to execute the appropriate bidding and auction logic inside a collection of dedicated worklets associated with the buyer and seller domains.  The `auctionSignals`, `sellerSignals`, and `perBuyerSignals` values will be passed as arguments to the appropriate functions that run inside those worklets — the `auctionSignals` are made available to everyone, while the other signals are given only to one party.
 
+The optional `requestedSize` field recommends a frame size for the auction, which will be available to bidders in browser signals. This size should be specified in the same format as the sizes in the `adSizes` field of `joinAdInterestGroup`. For convenience, the returned fenced frame config will automatically populate a `<fencedframe>`'s `width` and `height` attributes with the `requestedSize` when loaded, though the element's size attributes can still be modified if you want to change the element's container size. Bidders inside the auction may pick a different content size for the ad, and that resulting size will be visually scaled to fit inside the element's container size.
```

Comment:
So one of the results of using this in a FF will be that the ff.width and ff.height will implicitly be set to the requestedSize w/h. Will the same be true in iframes using the opaque URN? We generally seem to be keeping the same API, want to confirm.

Reply:
Also w/r/t:

_Bidders inside the auction may pick a different content size for the ad, and that resulting size will be visually scaled to fit inside the element's container size._

Is the theory that the API should allow this, but the seller can disallow it as a business choice via scoreAd?

Reply:
No, iframes with opaque urns don't respect respectedSize (or renderSize); they don't have any of the size restrictions that fenced frames do. In that case the only useful thing is the ability to macro renderSize into the ad creative url. But for now you could do that with deprecatedReplaceInURN.

And yes, the seller can disallow it if they want. It seems like you'd only want to do it as a fallback, since visual scaling obviously is unideal.